templates/membership/join.html.twig

```diff
@@ -98,6 +98,10 @@
                                 {{ form_errors(form.password) }}
                             </div>
                         {% endif %}
+                        <div class="l__row l__row--h-stretch register__form--full">
+                            {{ form_errors(form.nationality) }}
```

Comment:
in this template, the order of block is: `widget, errors`
but you display errors first and widget after. there is a real reason for that?

Reply:
reversed